php/libraries/NDB_Client.class.inc

```diff
@@ -133,15 +133,25 @@ class NDB_Client
             . $config_additions
         );
         // start php session
```

Comment:
ini_set('session.cookie_httponly', '1');
ini_set('session.use_strict_mode', '1');
session_start();

Reply:
For the session.cookie_secure that failed in Docker test suits, we can't modify the apache server "Add following entry in httpd.conf
Header edit Set-Cookie ^(.*)$ $1;HttpOnly;Secure"

Reply:
We should modify e.g. Docker (probably Vagrant) to include these settings server-side. However for users who do not install users this way it's better to use the application code to guarantee security rather than rely on the server configuration.